src/Core/Utilities/HandlebarsObjectJsonConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Bit.Core.Utilities
+{
+    public class HandlebarsObjectJsonConverter : JsonConverter<object>
+    {
+        public override bool CanConvert(Type typeToConvert) => true;
+        public override object Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            return JsonSerializer.Deserialize<Dictionary<string, object>>(ref reader, options);
```

Comment:
Can we no longer use `ExpandoObject`?

Reply:
We *could* but we would have to right our own custom converter and [Handlebars.Net](https://github.com/Handlebars-Net/Handlebars.Net#model-types) says a dictionary is even faster. The main difference between Newtonsoft.Json and System.Text.Json is that when it see's an `object` is deserializes it as `JsonElement` and `Newtonsoft.Json` did `dynamic`. So we could make our own but since handlebars likes `Dictionary<>` I think I prefer this.